### ISSUE TITLE
Overview table for paralipomena

### ISSUE THREAD

**Issue by gerritbruening** [faustedition/faust-web#134]
It would be good to make Paralipomena accessible by their numbers in WA and Bohnenkamp 1994.

Paralipomena are encoded in the following way:

     <milestone n="p78" spanTo="#p78_end" unit="paralipomenon"/>

Some have a numbering analogous to the normal one: `<l n="p78_1">`. Those which are in prose have no natural line numbering. We have used the line count in the fist edition (WA): `<milestone n="p1_1" unit="refline"/>`, but this was subject to criticism. Severe problems arise when there is no line-to-line matching as a result of strong variation (P123).

* [x] Generate a table of Paralipomena with the following colums:
 * (1) number
 * (2) idno of the document
 * (3) first words of the text
* discuss, specify and encode a line numbering
* Generate a line synopsis analogous to the one in the text view.

**Comment by thvitt** [comment]
Is the `<milestone unit="paralipomenon">` element always present, even for p. that have a l/@n style numbering?

**Comment by gerritbruening** [comment]
Yes, the milestone is always there!

**Comment by thvitt** [comment]
For the line synopsis, it should be possible to slightly enhance the current synopsis stuff when we have a consistent and parseable line numbering for paralipomena. Prose content could probably be treated similar to _Trüber Tag. Feld._ content. Should become a separate issue when the numbering scheme is there.

**Comment by gerritbruening** [comment]
Sounds reasonable. I will ask for current opinions about the line numbering.

**Comment by gerritbruening** [comment]
For the line synopsis and its prerequisite, the line numbering, see see #135.

**Comment by thvitt** [comment]
A first version of that table will appear in the next nightly of the dev version under [/paralipomena](http://dev.faustedition.net/paralipomena). There are still a few open issues:

* [ ] where should this be attached to the site structure? It is now in /archive, someone ≠ me should find a suitable position. @gerritbruening 
* [ ] table should be sortable. Table sort refactoring together with #94. @thvitt
* [ ] number of tokens for incipit. Currently it's 5. @gerritbruening 
* [ ] where should the links point to? @gerritbruening 
* [ ] we should probably work on the emended version. Blocked by faustedition/faust-gen-html#59 @thvitt
* [ ] additional cleanup in the incipit text (by adding rules [here](https://github.com/faustedition/faust-gen-html/blob/master/xslt/create-paralipomena-info.xsl#L148)) @gerritbruening

**Comment by gerritbruening** [comment]
Great!
Two small remarks:
* Column Paralipomenon should be Nummer WA
* ſ should be rendered as s

As to your questions:

1. where: The table should be attached to text.
2. number of tokens: I think five is OK. Ideally, <speaker> would be excluded which now seems to appear threefold: “Meph. . Meph.Meph.”
3. H P6 could still point to where it now points to, but “Nummer WA” / 6 should point to the line before xml/document/paralipomena/gsa_390782.xml&page=1&view=print#lp6_1 (i.e., text view, first chunk element after the respective `milestone`).
4. additional cleanup, see 2.

**Comment by gerritbruening** [comment]
Paralipomenon 2 (391373.xml) is missing (`<milestone unit="paralipomenon" n="p2" spanTo="#p2_end"/>`)

**Comment by thvitt** [comment]
The key is broken :(

**Comment by thvitt** [comment]
* Gerrit Brüning:
>   * ſ should be rendered as s

… after we redesigned this to build on the emended version this will be
the case

>  2. number of tokens: I think five is OK. Ideally, would be excluded
>     which now seems to appear threefold: “Meph. . Meph.Meph.”

If this is not due to editing markup that disappears in the emended
version, you'll need to  specify this at the encoding level (e.g., 'skip
`<speaker>`')

**Comment by gerritbruening** [comment]
What does "key is broken" in https://github.com/faustedition/faust-web/issues/134#issuecomment-227461996 mean?

Sorry, I forgot the gravis accents in my previous comment. But yes, what I meant was that `<speaker>` should be excluded

**Comment by thvitt** [comment]
* Gerrit Brüning:
> What does "key is broken" in #134 (comment)
> <https://github.com/faustedition/faust-web/issues/134#issuecomment-227461996>
> mean?

my xsl:key statement doesn't work as intended, that's why a lot of paralipomena are missing from the table (eg the one you mentioned)

And now for something completely different: My students' homework …

**Comment by gerritbruening** [comment]
Very good!
But I still see duplicates of text segments such as "Meph.Meph."

**Comment by gerritbruening** [comment]
What I see in the XML source is 

                    <speaker n="before_p6_1">
                        <abbr>Meph.</abbr>
                    </speaker>

Where can I influence the processing of these elements?

**Comment by thvitt** [comment]
As I said some comments ago, you'll have to check the sources of the paralipomena which have duplicates and identify which elements should be dropped. (The corresponding code is at the end of `xslt/create-paralipomena-info.xsl`, in `mode="text"`, but I'm ready to implement this, I just won't decide on the elements that should be dropped).
